src/properties/property-effects.html

```diff
@@ -1176,6 +1400,60 @@
       }
 
       /**
+       * Enqueues the given client on a list of pending clients, whose
+       * pending property changes can later be flushed via a call to
+       * `_flushClients`.
+       *
+       * @param {Object} client PropertyEffects client to enqueue
+       * @protected
+       */
+      _enqueueClient(client) {
+        this.__dataPendingClients = this.__dataPendingClients || new Map();
+        if (client !== this) {
+          this.__dataPendingClients.set(client, true);
+        }
+      }
+
+      /**
+       * Flushes any clients previously enqueued via `_enqueueClient`, causing
+       * their `_flushProperties` method to run.
+       *
+       * @protected
+       */
```

Comment:
Moved from `batched-effects` to `property-effects`. Only change is to reset `__dataPendingClients` before iterating.